spec/dom-elements-pool-spec.coffee

```diff
@@ -0,0 +1,41 @@
+DomElementsPool = require '../src/dom-elements-pool'
+
+describe "DomElementsPool", ->
```

Comment:
Some tests around failure cases would be good, especially for such an important piece of infrastructure. I would suggest tests like:

* Free something that was already freed
* Free `undefined` and `null`
* Free a child and then free the parent and descendants
* Build `undefined` and `null`

If I can think of any others, I'll mention them.